apps/passport-server/src/services/telegramService.ts

```diff
@@ -521,24 +514,19 @@ export class TelegramService {
     // Edge case logic to handle routing people between bots
     if (this.anonBotExists()) {
       this.authBot.command("anonsend", async (ctx) => {
-        await ctx.reply(
```

Comment:
will it just ignore /anonsend in group chats? it still might make sense to reply with "Please message ZuRat"

Reply:
@rrrliu added that message back!